packages/landing/src/components/debug.tsx

```diff
@@ -109,52 +114,108 @@ export class Debug extends Component<{ map: maplibre.Map }, { lastFeatureId: str
     this.setSourceShown(target.checked);
   };
 
+  trackMouseMove(layerId: string): void {
+    const sourceId = `${layerId}_source`;
+    const layerFillId = `${sourceId}_fill`;
+    const map = this.props.map;
+
+    // const source = '';
+    let lastFeatureId: string | number | undefined;
+    map.on('mousemove', layerFillId, (e) => {
+      const features = e.features;
```

Comment:
Are we got an Id for each feature?

Reply:
I force added a featureId further down `f.id = id++`

Reply:
Oh, I just saw it.